src/index.jsx

```diff
@@ -26,10 +29,21 @@ const App = () => (
   </Provider>
 );
 
-const Unauthorized = () => "You are not authorized to access the gradebook for this course.";
+const Unauthorized = () => (
+  <div>
+    You are not authorized to access the gradebook for this course.
+    &nbsp;
+    <a
+      href={`${configuration.LMS_BASE_URL}/courses/${courseId}/instructor`}
```

Comment:
What if they don't have access to the instructor dashboard?

Reply:
Then they shouldn't have been going to the gradebook in the first place ;)

Reply:
But actually, I don't know. what would you suggest?

Reply:
I dunno, depends on who this message is for.  If it's primarily for course team members that somehow don't have access to this page, then your current message with a link back to the instructor dashboard is probably fine.